Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ruby:2.1
```

Comment:
This Dockerfile builds an image that contains just Discourse itself. It's common for projects that support building as docker images in this kind of simple way to include a Dockerfile.

As far as I understand, the purpose of discourse_docker is quite different: it's to let a user build and run their own, custom discourse docker image (which contains postgres, redis, etc).